config/webhook/manifests.yaml

```diff
@@ -24,26 +24,6 @@ webhooks:
     resources:
```

Comment:
Why keeps /mutate-workloads-kubeblocks-io-v1alpha1-instanceset

Reply:
All cleaned up.

This file is maintained and updated by controller-gen, it seems that it behaves weirdly when the webhook marker is removed.